test/gateway.test.js

```diff
@@ -11,7 +11,7 @@ describe('gateway', () => {
   const id = uuidv4()
 
   before(done => {
-    MongoClient.connect('mongodb://localhost/todos-app', (err, _db) => {
+    MongoClient.connect('mongodb://localhost/todos-app-test', (err, _db) => {
```

Comment:
I recommend using a `TEST_MONGODB_URI` environment variable. `dotenv` is powerful because it allows multiple developers to collaborate on a project without having to sync their system configurations beyond hard requirements, e.g. have Node, Mongodb, and a browser installed.

Reply:
Good point. Need to remember that going forward.